docker/scylla-sct/ubuntu/Dockerfile

```diff
@@ -38,4 +38,4 @@ stderr_logfile=/dev/stderr\n\
 stderr_logfile_maxbytes=0' > /etc/supervisord.conf.d/scylla-manager.conf && \
     echo "autostart=false" >> /etc/supervisord.conf.d/scylla-server.conf && \
     echo "autostart=false" >> /etc/supervisord.conf.d/scylla-housekeeping.conf && \
-    sed -i "\:/dev/stderr:d" /etc/bashrc
+    sed -i "\:/dev/stderr:d" /etc/bash.bashrc
```

Comment:
Is this line required? I can't see any `stderr` ref on `/etc/bash.bashrc`

Reply:
most likely you are right, hence i removed it